docs/source/glossary.rst

```diff
@@ -4,50 +4,61 @@ Glossary
 .. glossary::
 
   Node
-    Machine running all the components necessary for a working blockchain
-    (including the Validator, the REST API, at least one transaction processor,
-    and the PBFT algorithm itself). In this RFC, unless otherwise specified,
-    it can be assumed that *Node* refers to the PBFT component of the machine.
+    Virtual or physical machine running all the components necessary for a
+    working Sawtooth blockchain: a validator, an optional REST API, at least
+    one transaction processor, and the PBFT consensus engine.
 
     The `original PBFT paper <http://pmg.csail.mit.edu/papers/osdi99.pdf>`__
     uses the terms `server` or `replica` instead of `node`.
 
   Validator
-    Component of a node responsible for interactions with the blockchain.
-    Interactions with the validator are abstracted by the Consensus API.
+    Sawtooth component that is responsible for interactions with the
+    blockchain. The validator interacts with the consensus engine through the
+    consensus API.
 
   Block
-    A part of the
-    `blockchain <https://en.wikipedia.org/wiki/Block chain>`__,
-    containing some operations and a link to the previous block.
+    Part of the `blockchain <https://en.wikipedia.org/wiki/Block chain>`__
+    that contains one or more operations (called `transactions`) and a link to
+    the previous block.
+
+  Primary node
+    Node that directs the consensus process for the network. (The other nodes
+    in the network participate as :term:`secondary nodes <secondary node>`.)
+    The primary node receives each block from the consensus API, publishes it
```

Comment:
This sentence isn't particularly accurate; try this: "The primary node creates and publishes each block to the network, and starts the consensus process for the block."

Reply:
Fixed.